src/duke/Deadline.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+
+public class Deadline extends Task {
```

Comment:
Maybe add JavaDoc for this public class can make your program clearer?

Reply:
Yes public classes should have java docs if possible